### PR TITLE
Hamburger

### PR DESCRIPTION
**Pull request by ahadb802**
Added Hambuger menu to mobile version

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -103,6 +103,13 @@ button:hover {
...
   border-radius: 0;
   padding: 24px 155px 24px 155px;
+  z-index: 100;
+}
+
+.sticky {
+  position: fixed;
+  top: 0;
+  width: 100%;
 }
```
Comment (christianbiring1) [inline stylesheet.css:114]: [OPTIONAL]
- [ ] For a better user experience, Kindly make sure the page width occupies the entire width of the screen on the mobile version. there's no need to have a scroll on the right.

Your design on mobile version `width: 375px` :point_down: 
image

image

File: stylesheet.css [hunk 2]
```diff
@@ -949,8 +956,46 @@ button:hover {
...
+  }
+
+  #menuBox {
+    width: 0;
+    height: 0;
+    background: rgba(7, 18, 114, 0.9);
+    position: fixed;
+    top: 0;
+    right: 0;
+    border-bottom-left-radius: 100%;
```
Comment (christianbiring1) [inline stylesheet.css]: - [ ] Kindly make sure you stick to the design as much as possible. You may consider using the some color, typo and spacing between your elements.
Your design :point_down: 
image

Figma template :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @ahadb802 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title`
- [ ] Good commit history.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [x] `I HIGHLY RECOMMEND YOU IMPLEMENT THEM FOR THEY WILL TAKE YOUR PROJECT TO THE NEXT LEVEL` :rocket: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Giving your PR a descriptive title.
`Hint: Mobile menu would sound great than hamburger`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -15,25 +15,28 @@
...
+        <div class="header" id="myHeader">
         <div class="logo">
         <a href="#">A.Ahad</a>
         </div>
-        <nav >
+        <img src="./images/Icon-Menu.png" alt="hambuger-menu" id="menuIcon" class="menu-icon">
+        <nav id="menuBox">
             <ul>
-                <li><a href="#">Portfolio</a>
+                <li class="nav-list">
```
Comment (topeogunleye) [inline index.html:25]: - [ ] As suggested by the previous reviewer, I kindly suggest that you stick to the design as much as possible. Please kindly add the background blur property that show when the mobile-menu is active. you can use Javascript to add and remove the 'bured' css class applied.

```
.blured {
  filter: blur(2px);
  -webkit-filter: blur(2px);
}
```

| Your Design  | Figma Design |
| ------------- | ------------- |
| image | image |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @ahadb802,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* Professional README file. :heavy_check_mark:
* PR has a good description :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- [ ] Kindly update your linters.yml file for this project to include ESLinters. In the root project, you have a folder named .github. Inside this one, you have another folder named workflows. And inside the workflows folder, you have a file named linters.yml. Inside this file, you need to update the linters. The linters needed for this project can be found [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js);
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @ahadb802     :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**